--- a/vendor/github.com/hyperledger/fabric-lib-go/bccsp/pkcs11/conf.go
+++ b/vendor/github.com/hyperledger/fabric-lib-go/bccsp/pkcs11/conf.go
@@ -1,10 +1,10 @@
 // PKCS11Opts contains options for the PKCS11 provider
 type PKCS11Opts struct {
-	Security int    `json:"security"`
-	Hash     string `json:"hash"`
+	Security int    `json:"security" yaml:"Security"`
+	Hash     string `json:"hash" yaml:"Hash"`
 
 	// PKCS11 options
-	Library        string         `json:"library"`
-	Label          string         `json:"label"`
-	Pin            string         `json:"pin"`
+	Library        string         `json:"library" yaml:"Library"`
+	Label          string         `json:"label" yaml:"Label"`
+	Pin            string         `json:"pin" yaml:"Pin"`
 }
